cli/src/config.rs

```diff
@@ -222,4 +223,19 @@ mod tests {
         let config: Config = parse_config(&test_config_file).unwrap();
         assert_eq!(config.request_type, PackageType::Npm);
     }
+
+    #[test]
+    fn test_pass_api_key_through_env() {
+        write_test_config();
+        let temp_dir = temp_dir();
+        let test_config_file = temp_dir.as_path().join("test_config");
+        env::set_var("PHYLUM_API_KEY", "ENV VARIABLE TOKEN");
```

Comment:
I'd suggest extracting the `"ENV VARIABLE TOKEN"` string into a variable so it can be reused for comparison later. That should make it easier to change in the future in case a random token string isn't valid anymore.

```suggestion
        let token = "ENV VARIABLE TOKEN";
        env::set_var("PHYLUM_API_KEY",  token);

        write_test_config();
        let temp_dir = temp_dir();
        let test_config_file = temp_dir.as_path().join("test_config");
```